cmd/subcommand.go

```diff
@@ -5,13 +5,16 @@ import (
 	"fmt"
 	"io"
 	"os"
+	"strings"
 
 	"github.com/lets-cli/lets/commands/command"
 	"github.com/lets-cli/lets/config"
 	"github.com/lets-cli/lets/runner"
 	"github.com/spf13/cobra"
 )
 
+const genericCmdTplPlaceholder = "${LETS_COMMAND_NAME}"
```

Comment:
It would be nice to add LETS_COMMAND_NAME env to the command environment as well. It may be useful to use it in `cmd` scripts.

Reply:
done